tasks/CarryMyLuggage.tex

```diff
@@ -33,11 +33,12 @@ \subsection*{Additional rules and remarks}
 \begin{enumerate}[nosep]
 	\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied as follows.
 	\begin{itemize}[nosep]
-		\item Handing-over the bag (-100 pts).
-		\item Finding the operator while following her
+		\item Handing the bag over (-100 pts).
+		\item Finding the operator while following them:
 		\begin{itemize}[nosep]
 		\item Natural interaction (e.g.~wave and calling) 100pts score reduction.
 		\item Non-natural interaction (e.g.~raising both hands and jumping) 200pts score reduction.
+		\item Moving back (e.g.~ask the operator to step back in front of the robot) 300pts score reduction.
```

Comment:
Why does this request get more score reduction than non-natural interactions? I think it can just count as a non-natural interaction.

Reply:
In the past it was straight out forbidden for the operator to move back whereas requesting to raise both arms was fine. So I thought it should be penalized more heavily. What do the others think? @RoboCupAtHome/technical-commmittee

Reply:
A distinction between natural and moving-back makes a lot of sense to me; I would rather clap than have to walk back because effort. Distinction between non-natural and moving back is less clear; I think I would rather move back than jump up and down, but raising both hands sounds easier than walking... So it may fit fine in that category, but the name would have to better describe what is implied; more-than minimal user effort required